main.go

```diff
@@ -177,6 +178,9 @@ func addLicense(path string, fmode os.FileMode, tmpl *template.Template, data *c
 		lic = append(line, lic...)
 	}
 	b = append(lic, b...)
+	if verbose {
+		log.Printf("%v modified", path)
```

Comment:
Go usually invites to use `%s` for strings.

Reply:
Done.